packages/components/src/DataList/DataList.css

```diff
@@ -50,6 +50,12 @@
   position: relative;
   padding: var(--space-small);
   border-bottom: var(--border-base) solid var(--color-border);
+  transition: all var(--timing-base);
+}
+
+.listItem.active {
+  --data-list-item-active-color: rgb(243, 245, 246);
```

Comment:
I just looked up this specific value and it seems _not that far_ from our `surface--background` - is this color just being pulled from the Figma?

Reply:
Yup!! I was like, "that does not exist. Chris will comment on it, let's push it" haha